source/reference/mongosync/mongosync-behavior.txt

```diff
@@ -140,16 +140,16 @@ Consistency
 ~~~~~~~~~~~
 
 ``mongosync`` supports :term:`eventual consistency` on the destination
-cluster. Read Consistency is not guaranteed on the destination cluster,
-the source and destination clusters may differ at a given point in
-time. However, if new writes are halted on the source cluster, the
-source and destination clusters will eventually hold the same data.
+cluster. Read consistency is not guaranteed on the destination cluster until 
+commit. Before committing, the source and destination clusters may differ at a 
+given point in time. However, if new writes are halted on the source cluster, 
+the source and destination clusters will eventually hold the same data.
 
 Individual writes may be reordered or combined as they are relayed from
```

Comment:
For this section starting with "Individual writes" and the below section starting with "Transactions might not", these are specifically for pre-commit mongosync, right? Can we label this more clearly? CC @rohshar

Reply:
After commit, mongosync won't be doing any other writes, so just the client should be doing the writes.

Reply:
updated.